annotations/src/main/java/org/robolectric/annotation/SecurityMode.java

```diff
@@ -0,0 +1,25 @@
+package org.robolectric.annotation;
```

Comment:
Make sure to run java format:

https://github.com/robolectric/robolectric/wiki/Running-google-java-format